Application: Added Categories Functionality

MuhammadAbdullah191 opened the pull request:
-> Added Category Model
-> Added Catrgory Controller
-> Added Admin ability to create, edit and update categories

MuhammadAbdullah191 committed:
Application: Added Categories Functionality
-> Added Category Model
-> Added Catrgory Controller
-> Added Admin ability to create, edit and update categories

husnainrazadevsinc commented on app/controllers/categories_controller.rb:
```diff
@@ -0,0 +1,54 @@
+  def destroy
+    @category.destroy
+    flash[:success] = 'Category Deleted successfully'
+    redirect_to categories_path
```
please take care of destroy failure case.

husnainrazadevsinc reviewed (changes requested):
Suggested a change, please check

MuhammadAbdullah191 committed:
Feedback Addressed:
-> Added Dafault value to categories model
-> Added blank check to category
-> Added Failure check to categories delete action
